reconcile/utils/jjb_client.py

```diff
@@ -383,8 +383,10 @@ def get_repo_url(job):
         repo_url_raw = job['properties'][0]['github']['url']
         return repo_url_raw.strip('/').replace('.git', '')
 
-    def get_all_jobs(self, job_types=[''], instance_name=None,
+    def get_all_jobs(self, job_types=None, instance_name=None,
                      include_test=False):
+        if not job_types:
+            job_types = []
```

Comment:
Just pointing out that this was previously a list with an empty string, but it looks like that probably doesn't matter?

Reply:
Good catch and indeed I did look at the code and in places where `job_types` is used I don't see how it would matter that the "default" list has a single empty string element

Reply:
Figured you caught it, but changes like these can make it easy to miss one outlier. Thanks!